modules/core/keeper/keeper.go

```diff
@@ -49,13 +51,23 @@ func NewKeeper(
 	connectionKeeper := connectionkeeper.NewKeeper(cdc, key, paramSpace, clientKeeper)
 	portKeeper := portkeeper.NewKeeper(scopedKeeper)
 	channelKeeper := channelkeeper.NewKeeper(cdc, key, clientKeeper, connectionKeeper, portKeeper, scopedKeeper)
+	wasmKeeper := wasmkeeper.NewKeeper(cdc, key, &wasmkeeper.VMConfig{
+		DataDir:           "wasm_data",
+		SupportedFeatures: []string{"staking"},
+		MemoryLimitMb:     8,
+		PrintDebug:        true,
+		CacheSizeMb:       8,
+	}, &wasmkeeper.ValidationConfig{
+		MaxSizeAllowed: 1024 * 1024, // 1 MB
+	})
```

Comment:
Is there a way to read/accesss configuration from keeper.go? Right now I left hardcodes, but this definitely should be configurable

Reply:
It probably needs to be passed in as an argument from the app.go file

Who is doing the configuring here? This is a chain level parameter correct? Any reason for it not to be a parameter controlled by governance?

Reply:
Good question. I would propose:

1. Governance controlled:
   * MemoryLimitMb
   * MaxSizeAllowed
 
2. Node/Validator config:
  * DataDir
  * CacheSizeMb
  * PrintDebug

I would leave `SupportedFeatures` empty and hardcoded. Is there an usecase for Light Client to use other modules?:
```
SupportedFeatures: []string{},
```